dao/src/main/java/org/thingsboard/server/dao/user/UserServiceImpl.java

```diff
@@ -326,6 +328,27 @@ public int increaseFailedLoginAttempts(TenantId tenantId, UserId userId) {
         return failedLoginAttempts;
     }
 
+    @Override
```

Comment:
Move this method to UserSettingsService. Make UserSettingsService extend `AbstractCachedService` (similar to AbstractCachedEntityService)

Reply:
See AssetRedisCache and AssetCaffeineCache

Reply:
and also update thingsboard.yml to add new cache